server/src/test/java/org/opensearch/cluster/metadata/MetadataCreateIndexServiceTests.java

```diff
@@ -817,7 +817,8 @@ public void testAggregateSettingsAppliesSettingsFromTemplatesAndRequest() {
             Settings.EMPTY,
             IndexScopedSettings.DEFAULT_SCOPED_SETTINGS,
             randomShardLimitService(),
-            Collections.emptySet()
+            Collections.emptySet(),
+            false
```

Comment:
Lets please add unit tests here on `aggregateIndexSettings` to validate new logic in `updateReplicationStrategy`, both with/without sys index and hidden.  We should also be able to validate the output replication type setting at cluster/index level.

Reply:
added in latest commit